build.gradle.kts

```diff
@@ -61,9 +61,9 @@ dependencies {
   developmentOnly("org.springframework.boot:spring-boot-devtools")
 
   testImplementation("org.awaitility:awaitility-kotlin")
-  testImplementation("io.jsonwebtoken:jjwt-impl:0.11.5")
-  testImplementation("io.jsonwebtoken:jjwt-jackson:0.11.5")
-  testImplementation("org.mockito:mockito-inline")
+  testImplementation("io.jsonwebtoken:jjwt-impl:0.12.0")
+  testImplementation("io.jsonwebtoken:jjwt-jackson:0.12.0")
+  testImplementation("org.mockito:mockito-inline:5.2.0")
```

Comment:
version of `org.mockito:mockito-inline` was unspecified and tests seemed to fail with errors mentioning this dependency… unclear what the issue was 🤷